src/electionguard/words.py

```diff
@@ -4,10 +4,11 @@
 
 def get_word(index: int) -> str:
     """
-    Get word(4096 options) based on 16 bit index for use with trackers.
+    Get word (4096 options) based on 16 bit index for use with trackers.
     :param index: index of word
     :return: word
     """
+    # TODO: should this generate a warning if the index is out of range?
```

Comment:
Since we're public, we need to keep `TODOs` to an affirmative one line task and create issues accordingly to the todos. This seems more like a inquisitive thought than a firm `TODO`

Reply:
keith didnt you already capture this somewhere else?  i know it was a known defect at one point